mwapi/session.py

```diff
@@ -39,15 +41,21 @@ def __init__(self, host, api_path="/w/api.php", session=None):
         self.api_path = api_path
         self.api_url = host + api_path
         self.session = session or requests.Session()
-        self.is_authenticated = False
-        """
-        `bool` : Is the session sending authenticated requests
-        """
 
     def _request(self, method, params=None, data=None, files=None):
         resp = self.session.request(method, self.api_url, params=params,
                                     data=data, files=files, stream=True)
-        return resp.json()
+
+        try:
+            doc = resp.json()
```

Comment:
I always thought that "doc" as in json document was intuitive.  "result" could be all sorts of different types of things.  I'm down with spelling out "document" completely, but that seems verbose when "doc" was assumed to stand for "document" in the first place.    See "token_doc" and "login_doc" below.